primo/utils/override_utils.py

```diff
@@ -112,6 +112,13 @@ def assess_owner_well_count(self) -> Dict:
         Returns list of owners and wells selected for each for whom the owner
         well count constraint is violated
         """
+        opt_inputs = self.opt_inputs.config
+        max_wells_per_owner = opt_inputs.max_wells_per_owner
+        if max_wells_per_owner is None:
+            # When the user does not have owner information
+            # or does not wish to prioritize
+            # this constraint becomes meaningless
+            return 0
```

Comment:
Please ensure return type for a method is consistent across all branches. Since the return type for this method is a Dict, perhaps output in this case can be {}?